src/utils/helpers/browser/browser.js

```diff
@@ -11,6 +10,15 @@ import Form from './../../../components/form';
 *
 */
 const Browser = {
+  isDomAvailable: () => {
+    const _window = Browser.getWindow();
+    const _document = Browser.getDocument();
+    return (!!(
+      typeof _window !== 'undefined' &&
+      _document &&
+      _document.createElement
```

Comment:
Could you use `_document.hasOwnProperty('createElement')` here instead of the `!!`? This is a question rather than a request to change. There isn't really any difference in them

Reply:
Think the `!!` is still required to ensure it is a boolean if window or document are undefined.

To be honest this code was taken from the third party lib we used to use https://github.com/tajo/react-portal/blob/master/src/utils.js